src/library-authoring/import-course/index.scss

```diff
@@ -0,0 +1,29 @@
+.course-migration-help {
+  z-index: 1000; // same as header
+  flex: 350px 0 0;
+  position: sticky;
+  top: 0;
+  right: 0;
+  height: 100vh;
+  overflow-y: auto;
+
+  hr {
+    width: 100%;
+  }
+}
```

Comment:
We can remove this completely. It unnecessarily adds vertical scroll

Reply:
I was trying to avoid the same issue from here: https://github.com/openedx/frontend-app-authoring/issues/2200#issuecomment-3411649267

Removed: e11f76f0499f7d7f44644b8809f764ac34a47aae